code/datums/quirks/negative_quirks/claustrophobia.dm

```diff
@@ -12,7 +12,7 @@
 	quirk_holder.clear_mood_event("claustrophobia")
 
 /datum/quirk/claustrophobia/process(seconds_per_tick)
-	if(quirk_holder.stat != CONSCIOUS || quirk_holder.IsSleeping() || quirk_holder.IsUnconscious())
+	if(quirk_holder.stat != CONSCIOUS || quirk_holder.IsSleeping() || quirk_holder.IsUnconscious() || HAS_TRAIT(quirk_holder, TRAIT_MIND_TEMPORARILY_GONE))
```

Comment:
could you just make this a separate check, this if statement is stupidly long

Reply:
22c0af086d7447ca0051a345ce6d8fbbaa16c7f2